src/vivarium_testing_utils/automated_validation/data_loader.py

```diff
@@ -52,7 +54,7 @@ def get_sim_outputs(self) -> list[str]:
         return [str(f.stem) for f in self._results_dir.glob("*.parquet")]
 
     def get_artifact_keys(self) -> list[str]:
-        return self._artifact.keys
+        return self._artifact.keys  # type: ignore[no-any-return]
```

Comment:
This is appropriately type-hinted [here](https://github.com/ihmeuw/vivarium/blob/main/src/vivarium/framework/artifact/artifact.py#L56). We shouldn't need this ignore.